playwright/pages/siteSettingsPage.ts

```diff
@@ -58,9 +63,11 @@ export class SiteSettingsPage extends BasePage {
     await this.addressLine2Input.fill(addressLine2);
     await this.city.fill(city);
     await this.countryInput.click();
-    await this.autocompleteDropdown.getByText(country, { exact: true }).click();
+    await this.autocompleteDropdownCountry.getByText(country, { exact: true }).click();
+    await this.autocompleteDropdownCountry.blur();
     await this.countryAreaDropdown.fill(countryArea);
-    await this.autocompleteDropdown.getByText(countryArea, { exact: true }).click();
+    await this.autocompleteDropdownCountryArea.getByText(countryArea, { exact: true }).click();
+    await this.autocompleteDropdownCountryArea.blur();
```

Comment:
question: why did you change those selectors?

Reply:
Before this change there was a single selector for two dropdowns implying that all dropdowns have one id. This test was failing because Combobox doesn't pass test id in a way this test expected it to. I decided to make separate selectors for these two dropdowns and pass id so that we can select it using 'data-portal-for' - https://github.com/saleor/macaw-ui/commit/65a8f3f0a7e56e7f5a4b9ee2e0fd391f5e144ce0. 
I think this test wasn't checked after migration to Combobox.